### ISSUE TITLE
Same caption for all images, finetuning GIT

### ISSUE THREAD

**Issue by ArminBaz** [NielsRogge/Transformers-Tutorials#247]
Hi there, 

Thanks for adding GIT to Hugging Face! I'm currently finetuning it on my own custom dataset. However, after finetuning for an epoch or two I noticed that the model seems to output the same caption for all of my images. As I'm relatively new to image captioning I was hoping that you might be able to provide me with some assistance as to why this could be occurring.

My captions are all very similar, with some slight differences based on the road conditions in an image. My vocabulary size is also quite small (O, E, |, ||). I haven't changed anything with regards to the processor or the model, so I'm sure that could be a part of the issue as well.

Any and all help would be greatly appreciated.

Thanks!

**Comment by Sino0904** [comment]
I am facing the same issue. After fine-tuning all input images, the model generates the same text as the output. 
@NielsRogge

**Comment by ArminBaz** [comment]
@Sino0904 I resolved this issue by using a different learning rate than that in the tutorial. Try lowering it to 1.5e-5 - 1.5e-6 and see how that works. 

I’m keeping this open in case they want to comment on it.
